packages/Select/Select.jsx

```diff
@@ -279,6 +283,10 @@ Select.propTypes = {
    * Clarify the expected input.
    */
   hint: PropTypes.string,
+  /**
+   * Position of the `hint` relative to `label`.
+   */
```

Comment:
Could you also add `@since x.x.x` and specify which version this prop is available since?

Reply:
Added.